test/unit_tests/service/season/season_service_test.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:f1_application/lib/model/season.dart';
+import 'package:f1_application/lib/service/season/season_service.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  final SeasonService service = SeasonService();
+
+  final List<Season> seasons = [
+    Season(year: "1991", wikiURL: "https://wikipedia.com", driverIds: []),
+    Season(year: "1992", wikiURL: "https://wikipedia.com", driverIds: []),
+    Season(year: "1993", wikiURL: "https://wikipedia.com", driverIds: []),
+    Season(year: "1994", wikiURL: "https://wikipedia.com", driverIds: []),
+    Season(year: "1995", wikiURL: "https://wikipedia.com", driverIds: []),
+  ];
+
+  group("Season Service test(s)", () {
+    group("sortSeasonListDescByYear function test(s)", () {
+      service.sortSeasonListDescByYear(seasons);
+
+      test("After short the first season is 1995", () {
+        expect(seasons.first.year == "1995", true);

```

Comment:
Csak stilisztikai dolog: visszatérően több helyen így assert-elsz ahogy a kijelölt kódsorban, de lehet így is :

```
expect(seasons.first.year, "1995");
```


Ez lenne a preferált azért, mert így jobb lesz az output ha esetleg nem egyezik a két érték, például: 

```
expect(seasons.first.year == "1996", true);

-> 

test/unit_tests/service/season/season_service_test.dart 21:9  main.<fn>.<fn>.<fn>

Expected: <true>
  Actual: <false>
```

vs. 

```
test/unit_tests/service/season/season_service_test.dart 21:9  main.<fn>.<fn>.<fn>

Expected: '1996'
  Actual: '1995'
   Which: is different.
          Expected: 1996
            Actual: 1995
                       ^
           Differ at offset 3
```

Reply:
Okés! Át is írom!!! Ez így sokkal jobb, ahogy írtam úgy nem derül ki, hogy milyen év van rögzítve.